client/components/game-forms/CreateGameForm.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import {Field, reduxForm} from 'redux-form'
+import propTypes from 'prop-types'
+
+let CreateGameForm = props => {
+  const {handleSubmit, pristine, submitting} = props
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="eventName">Event Name </label>
+      <Field name="name" component="input" type="text" placeholder="My Event" />
+      <label htmlFor="date">Date of Event</label>
+      <Field
+        name="date"
+        component="input"
+        type="date"
+        placeholder={Date.now()}
+      />
+      <label htmlFor="location"> Event Location</label>
+      <Field
+        name="location"
+        component="input"
+        type="text"
+        placeholder="Chicago"
+      />
+      <label htmlFor="description">Event Description</label>
+      <Field
+        name="description"
+        component="input"
+        type="text"
+        placeholder="An Event in Chicago"
+      />
+      <button type="submit" disabled={pristine || submitting}>
+        {' '}
+        Create Game{' '}
+      </button>
+    </form>
+  )
+}
+
```

Comment:
Eventually this should include an initialValues prop.  It can be defined here or in a higher level component.